community/qca/APKBUILD

```diff
@@ -1,37 +1,34 @@
-# Contributor: 
+# Contributor: Leo <thinkabit.ukim@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=qca
-pkgver=2.1.3
-pkgrel=6
+pkgver=2.2.1
+pkgrel=0
 pkgdesc="Qt cryptographic architecture"
 url="https://userbase.kde.org/QCA"
 arch="all"
-license="LGPL"
-depends_dev="qt5-qtbase-dev"
-makedepends="$depends_dev cmake cyrus-sasl-dev"
-install=""
-subpackages="$pkgname-dev $pkgname-doc"
-source="http://download.kde.org/stable/qca/$pkgver/src/qca-${pkgver}.tar.xz
-	qca-openssl-1.1.patch
-	"
-
-builddir="$srcdir"/qca-$pkgver
+license="LGPL-2.1-or-later"
+makedepends="qt5-qtbase-dev cmake cyrus-sasl-dev"
```

Comment:
Why did you remove `qt5-qtbase-dev` from `-dev` subpackage?

Reply:
it wasn't removed, it now tracks via pc: provider. It actually tracked before but nobody cared to fix it.

```
qca-dev-2.2.1-r0 depends on:
libqca=2.2.1-r0
pc:Qt5Core
pkgconfig

qca-dev-2.1.3-r6 depends on:
qt5-qtbase-dev
pc:Qt5Core
pkgconfig
qca=2.1.3-r6
```